test/unit/instanceAndUpdate.spec.js

```diff
@@ -139,6 +140,26 @@ describe('instantiating a structure', () => {
         expect(user.name).to.equal('Not the default');
       });
     });
+
+    describe('instantiating a structure with buildStrict', () => {
+      context('when object is invalid', () => {
+        it('throw an error', () => {
+          expect(() => {
+            User.buildStrict();  
+          }).to.throw(Error, 'Invalid Attributes');  
```

Comment:
I think we can make this test more detailed using the `with.property` assertion as well, like this:

```js
expect(badFn).to.throw(TypeError).with.property('code', 42);
```

This is not the actual code we'd use in the test, I took this example from here https://www.chaijs.com/api/bdd/#method_throw. We could use it to test the `details` as well. What do you think?